### PR TITLE
fix: handle chained TimeoutException

### PR DESCRIPTION
**Pull request by sverrehu**

### CODE REVIEW

File: src/main/java/io/statnett/k3a/lagexporter/ClusterLagCollector.java [hunk 1]
```diff
@@ -231,4 +237,18 @@ private void invalidateClients() {
         }
     }
 
+    private static boolean isTimeout(final Throwable t) {
+        Throwable current = t;
+        while (current != null) {
+            if (TimeoutException.class.isAssignableFrom(current.getClass())) {
+                return true;
+            }
+            if (current == current.getCause()) {
```
Comment (erikgb) [inline src/main/java/io/statnett/k3a/lagexporter/ClusterLagCollector.java:246]: Is this required? Looks a bit suspicious.

Comment (sverrehu) [reply src/main/java/io/statnett/k3a/lagexporter/ClusterLagCollector.java:246]: The `TimeoutException` we are looking for sometimes gets wrapped in an `ExecutionException`.

Comment (erikgb) [reply src/main/java/io/statnett/k3a/lagexporter/ClusterLagCollector.java:246]: Strange...

**Review by erikgb — APPROVED** [review]
LGTM

**Pull request merged**